cli/Sources/TuistSupport/Extensions/AbsolutePath+Extras.swift

```diff
@@ -68,6 +68,7 @@ extension AbsolutePath {
             "bundle",
             "mlmodelc",
             "xcmappingmodel",
+            "icon",
```

Comment:
Should we have a test for it?

Reply:
Not really a need since we're not adding any business logic here. We could probably add a fixture, but keeping it simple for now